generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.ts

```diff
@@ -16,8 +16,7 @@ import { EventManager, AlertService<% if (enableTranslation) { %>, JhiLanguageSe
 import { <%= entityClass %> } from './<%= entityFileName %>.model';
 import { <%= entityClass %>PopupService } from './<%= entityFileName %>-popup.service';
 import { <%= entityClass %>Service } from './<%= entityFileName %>.service';
-<%- include('model-class-import-template.ejs'); -%>
```

Comment:
I thought it is simpler to import always the model and the service like this:

import { TournamentType, TournamentTypeService } from '../tournament-type/';
import { RankingList, RankingListService } from '../ranking-list/';